pydicom/charset.py

```diff
@@ -638,7 +638,7 @@ def _encode_string_impl(
 
 
 def convert_encodings(
-    encodings: Union[str, MutableSequence[str]]
+    encodings: Union[None, str, MutableSequence[str]]
```

Comment:
Every time I think I have the character set encoding figured out I find a new wrinkle